docs/learn/expert-guides/4337-extension.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_label: ' 🛠️ Integrating EIP-4337'
+description: 'Learn how to integrate the EIP-4337 extension into your Universal Profile.'
+sidebar_position: 9
+---
+
+# Integrating EIP-4337 into your Universal Profile
+
+:::info
+This guide assumes that you are already familiar with the [EIP-4337](https://eips.ethereum.org/EIPS/eip-4337) protocol and the [Universal Profile](../../standards/introduction.md) standard.
+:::
+
+## Prerequisites for Integration
+
+### Key Manager
+
+The Universal Profile contract must be owned by a [KeyManager](../../standards/universal-profile/lsp6-key-manager.md) contract.
+
+### Bundlers
+
+Bundlers are the key component of the EIP-4337 protocol. They are responsible for bundling `UserOperations` into transactions and submitting them to the EVM blockchain. You will need to run your own bundler or use a third-party service on your behalf. See the Ethereum implementation of their [bundler](https://github.com/eth-infinitism/bundler?tab=readme-ov-file#bundler).
+
+### Deposit on the EntryPoint contract
+
+The [`EntryPoint`](https://eips.ethereum.org/EIPS/eip-4337#required-entry-point-contract-functionality) contract is the entry point for the EIP-4337 protocol. It is responsible for receiving `UserOperations` and forwarding them to the appropriate account. In order to use the `EntryPoint` contract, you must first deposit a certain amount of native token on the contract. This deposit is used to pay the bundlers for their services.
+
+### Have a controller with the 4337 Extension Permission
+
+The controller of the Universal Profile that will use the 4337 extension must have the [`_4337_PERMISSION`](https://github.com/lukso-network/lsp-smart-contracts/blob/0a951df15f282840ef89499da94e6ab47380d5dd/contracts/LSP17Extensions/Extension4337.sol#L33) permission in order to use it. To learn more about permissions, refer to the [LSP6 standard](../../standards/universal-profile/lsp6-key-manager#permissions).
```

Comment:
I don't see any reference of this permission anywhere else.

It is not in erc725js, nor erc725 inspect and not in our docs:

<img width="1122" alt="image" src="https://github.com/lukso-network/docs/assets/477945/d88c08b5-8aa3-4989-87bb-e0325ac1e6e0">

https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager#permissions

Is it something we need to update? Or is it a "private" permission in some way?

Reply:
true this part is a bit confusing but it is an "internal" permission that is within the extension. Need to make it clearer

Reply:
I am not sure neither how we should go with this permission, and where we should add it. @frozeman might have some insights maybe?

Reply:
its specific to our lsp6 implementation, but it should be in erc725.js and added under `Payload Execution` imo